### ISSUE TITLE
`keybase track` fails on DNS tracking that doesn't use _keybase subdomain

### ISSUE THREAD

**Issue by kballard** [keybase/keybase-issues#1362]
I just tried to track andymatuschak using `keybase track andymatuschak`. The output looked like this:

```
> keybase track andymatuschak
info: ...checking identity proofs
✔ public key fingerprint: 289E CB4D BCC9 5D3A 71BD 6795 14CB 9F07 992C D348
✔ "andy_matuschak" on twitter: https://twitter.com/andy_matuschak/status/450064911071080448
✔ "andymatuschak" on github: https://gist.github.com/9865286
✔ "andymatuschak" on coinbase: https://coinbase.com/andymatuschak/public-key
✔ "andymatuschak" on hackernews: https://news.ycombinator.com/user?id=andymatuschak
✔ admin of andymatuschak.org via HTTP: http://andymatuschak.org/.well-known/keybase.txt
✖ admin of the DNS zone for andymatuschak.org (failed with code 107: queryTxt ENOTFOUND _keybase.andymatuschak.org)
Some remote proofs failed!
Still verify this user as andymatuschak? [y/N]
```

The problem is Andy verified his DNS using the TXT record of `andymatuschak.org`, not of `_keybase.andymatuschak.org`. Running `dig andymatuschak.org TXT` shows the expected verification.

**Comment by teh-maxh** [comment]
Make sure you're using the latest version of keybase (0.7.4). It works for me.

    $ keybase id andymatuschak
    ✔ public key fingerprint: 289E CB4D BCC9 5D3A 71BD 6795 14CB 9F07 992C D348
    ✔ "andy_matuschak" on twitter: https://twitter.com/andy_matuschak/status/450064911071080448
    ✔ "andymatuschak" on github: https://gist.github.com/9865286
    ✔ "andymatuschak" on coinbase: https://coinbase.com/andymatuschak/public-key
    ✔ "andymatuschak" on hackernews: https://news.ycombinator.com/user?id=andymatuschak
    ✔ admin of andymatuschak.org via HTTP: http://andymatuschak.org/.well-known/keybase.txt
    ✔ admin of the DNS zone for andymatuschak.org
    ฿ bitcoin 13A5d82zV1SYxkQvC5KzaYzdE9Xmfbzt6A

I do recommend using `_keybase.[domain]` instead of `[domain]` on its own, even though either method works.

**Comment by kballard** [comment]
I was using 0.7.3 as that was the latest version available 6 days ago.

**Comment by farcaller** [comment]
Reproducible at 0.7.4 too.

**Comment by farcaller** [comment]
Additionally, when I moved the recored to `_keybase.farcaller.net`, `keybase id` fails with:

`✖ admin of the DNS zone for farcaller.net (failed with code 201)`

**Comment by maxtaco** [comment]
I think this broke in Node v0.12.  Which version are you guys running?

**Comment by maxtaco** [comment]
Should be fixed in v0.7.5.  Let me know if not.  Thanks for the bug report. 

(BTW, this was a problem introduced with Node v0.12).
